budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/web/pages/user/selectproject/SelectProjectPage.java

```diff
@@ -50,9 +50,14 @@ public SelectProjectPage(Class<? extends WebPage> backlinkPage, PageParameters b
         Form<String> form = new Form<String>(id, new Model<String>("")) {
             @Override
             protected void onSubmit() {
-                ProjectBaseData project = projectService.createProject(getModelObject(), BudgeteerSession.get().getLoggedInUser().getId());
-                BudgeteerSession.get().setProjectId(project.getId());
-                setResponsePage(DashboardPage.class);
+                try {
+                    ProjectBaseData project = projectService.createProject(getModelObject(), BudgeteerSession.get().getLoggedInUser().getId());
+                    BudgeteerSession.get().setProjectId(project.getId());
+                    setResponsePage(DashboardPage.class);
+                }catch (Exception e){
```

Comment:
We're catching ALL exceptions here. What we really want to do is catch the `ProjectNameAlreadyInUseException `. When another exception occurs, it should not be caught (and thus moved up the call stack)